tomviz/ModuleContourWidget.cxx

```diff
@@ -0,0 +1,147 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+#include "ModuleContourWidget.h"
+#include "ui_LightingParametersForm.h"
+#include "ui_ModuleContourWidget.h"
+
+#include "vtkSMProxy.h"
+#include "vtkSMSourceProxy.h"
+
+#include "pqPropertyLinks.h"
+#include "pqSignalAdaptors.h"
+#include "pqWidgetRangeDomain.h"
+
+namespace tomviz {
+
+ModuleContourWidget::ModuleContourWidget(QWidget* parent_)
+  : QWidget(parent_), m_ui(new Ui::ModuleContourWidget),
+    m_uiLighting(new Ui::LightingParametersForm)
+{
+  m_ui->setupUi(this);
+
+  QWidget* lightingWidget = new QWidget;
+  m_uiLighting->setupUi(lightingWidget);
+  m_uiLighting->gbLighting->setCheckable(false);
+  QWidget::layout()->addWidget(lightingWidget);
+  qobject_cast<QBoxLayout*>(QWidget::layout())->addStretch();
+
+  m_ui->colorChooser->setShowAlphaChannel(false);
+
+  const int leWidth = 50;
+  m_ui->sliValue->setLineEditWidth(leWidth);
+  m_ui->sliOpacity->setLineEditWidth(leWidth);
+  m_uiLighting->sliAmbient->setLineEditWidth(leWidth);
+  m_uiLighting->sliDiffuse->setLineEditWidth(leWidth);
+  m_uiLighting->sliSpecular->setLineEditWidth(leWidth);
+  m_uiLighting->sliSpecularPower->setLineEditWidth(leWidth);
+
+  // TODO Ambient and diffuse are not playing well with coloring by data
+  // (non-solid color).
+  m_uiLighting->sliAmbient->hide();
```

Comment:
This may be due to an issue @tjcorona is tracking - contours in general are a little odd right now due to some changes made for categorical data. We are hoping to restore the original default, and then enable the changes for categorical coloring when using a label map.